lib/settings_base.py

```diff
@@ -1230,6 +1230,10 @@ def JINJA_CONFIG():
 CEF_PRODUCT = "amo"
 
 # CSP Settings
+
+PROD_CDN_HOST = 'https://addons.cdn.mozilla.net'
```

Comment:
I added it here because I see prod cdn requests in stage too. Also it's not entirely clear from just the hostname so I went for being very explicit.